test-app/.github/workflows/rename_template_project_pr.yml

```diff
@@ -0,0 +1,43 @@
+name: rename-project-create-pr
```

Comment:
do you need this for the test app?

Reply:
As discussed I'm just pulling everything in from the template to make it easier to see how the template has drifted in future.